src/framework/framework.cr

```diff
@@ -61,9 +61,9 @@ module Ktistec
     end
 
     def assign(options)
-      @host = options["host"] if options.has_key?("host")
-      @site = options["site"] if options.has_key?("site")
-      @footer = options["footer"] if options.has_key?("footer")
+      @host = options["host"].to_s if options.has_key?("host")
+      @site = options["site"].to_s if options.has_key?("site")
+      @footer = options["footer"].to_s if options.has_key?("footer")
```

Comment:
This was only working before because the values were all `String`s

Reply:
this is a good catch. i'd recommend `.as(String)` instead, which will ensure it has the right type but won't accidentally coerce some non-string value into a string.

Reply:
Good call